server/config.ts

```diff
@@ -30,6 +30,8 @@ export interface ApiConfig {
 }
 
 export default {
+  buildNumber: get('BUILD_NUMBER', '1_0_0', requiredInProduction),
+  gitRef: get('GIT_REF', 'xxxxxxxxxxxxxxxxxxx', requiredInProduction),
```

Comment:
blow up if env var not set when running the image in docker, default when running locally